paper/architectures/autoencoder.py

```diff
@@ -0,0 +1,49 @@
+import torch.nn as nn
+
+class Encoder(nn.Module):
+    
```

Comment:
Remove emtpy line

Reply:
and set up flake8 to detect some of this :) autoflake8 is also cool

Reply:
fixed in [adc9963](https://github.com/KRLGroup/memory-wrap/pull/3/commits/adc9963ccf57d23617e9f604ef3c98f47698807d)